### PR TITLE
Seasons

### PR DESCRIPTION
**Pull request by Ebitari-A**
Created seasons controller, seasons page with episodes showing
CSS to be added

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -33,8 +33,8 @@
   request_us["x-rapidapi-host"] = 'streaming-availability.p.rapidapi.com'
   request_us["x-rapidapi-key"] = ENV["RAPID_API_KEY"]
 
-  response_gb = JSON.parse(http_gb.request(request_gb).read_body)
-  response_us = JSON.parse(http_us.request(request_us).read_body)
+  p response_gb = JSON.parse(http_gb.request(request_gb).read_body)
```
Comment (alisonclare93) [inline db/seeds.rb:36]: Can you delete the p before these lines?

**Review by alisonclare93 — COMMENTED** [review]
Looks good! There's a couple of 'p's you could remove - I've left a comment on the line :)

**Comment by dimitripantelli** [comment]
Thumbs UP

**Review by alisonclare93 — COMMENTED** [review]
Looks good :) 👍

**Pull request merged**